pkg/notes/model/note.ts

```diff
@@ -14,6 +15,7 @@ export interface CreateNoteArgs {
   contentsWarningComment: string;
   sendTo: Option.Option<AccountID>;
   originalNoteID: Option.Option<NoteID>;
+  attachmentFileID: MediumID[];
```

Comment:
```suggestion
  attachmentFileID: readonly MediumID[];
```
`Note` の可変性にかかわらず, 利便性のためにここは `readonly` を付けておきたいです.